kclvm/api/src/service/ty.rs

```diff
@@ -47,6 +47,9 @@ pub(crate) fn kcl_schema_ty_to_pb_ty(schema_ty: &SchemaType) -> KclType {
                 ..Default::default()
             })
             .collect(),
+        filename: schema_ty.filename.clone(),
+        pkg_path: schema_ty.pkgpath.clone(),
+        description: schema_ty.doc.clone(),
```

Comment:
Also need to set the value of `ty.description` to attribute doc in `fn get_schema_ty_attributes`

Reply:
> Also need to set the value of `ty.description` to attribute doc in `fn get_schema_ty_attributes`

fixed.